sdk/forms/src/main/java/com/klaviyo/forms/InAppFormsConfig.kt

```diff
@@ -0,0 +1,10 @@
+package com.klaviyo.forms
+
+/**
+ * Configuration for in app forms
+ *
+ * @param sessionTimeoutDuration (seconds) timeout for listening to new in app forms to display
+ */
+data class InAppFormsConfig(
+    val sessionTimeoutDuration: Long
```

Comment:
Lets put a default value on here?

Reply:
My thinking is that (when this class supports more things) the developer should be able to flexibly specify  only the config values they care to override without having to set ALL the things.